ndex2/client.py

```diff
@@ -709,6 +943,18 @@ def get_task_by_id(self, task_id):
         """
         Retrieves a task by id
 
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		task_id = 'INSERT TASK ID HERE'
+		task = my_ndex.get_task_by_id(task_id)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client